static/service-worker-registration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
```

Comment:
Ok.

Then I suggest using `<script defer` because `<script async` may have fired after window load and makes service worker not register.